structs.go

```diff
@@ -325,9 +325,9 @@ type ChannelFollow struct {
 // A PermissionOverwrite holds permission overwrite data for a Channel
 type PermissionOverwrite struct {
 	ID    string `json:"id"`
-	Type  string `json:"type"`
-	Deny  int    `json:"deny"`
-	Allow int    `json:"allow"`
+	Type  int    `json:"type"`
```

Comment:
I know that we didn't have one before, but this seems like a great opportunity to define a constant in the library for easy comparison (rather than looking up the value in the Discord docs).